boards/arm/chester_nrf52840/dts/bindigs/skyworks,sky13335.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2023 HARDWARIO a.s.
```

Comment:
Please, use this format for the license identifier (as we have it everywhere else):

```
#
# Copyright (c) 2023 HARDWARIO a.s.
#
# SPDX-License-Identifier: LicenseRef-HARDWARIO-5-Clause
#
```

Reply:
fixed